src/coreclr/vm/arm64/stubs.cpp

```diff
@@ -1625,6 +1625,8 @@ void StubLinkerCPU::EmitCallLabel(CodeLabel *target, BOOL fTailCall, BOOL fIndir
 
 void StubLinkerCPU::EmitCallManagedMethod(MethodDesc *pMD, BOOL fTailCall)
 {
+    pMD->EnsureTemporaryEntryPoint(pMD->GetLoaderAllocator());
```

Comment:
Similar to the `arm/stubs.cpp` comment. A comment and contract seems appropriate here.